docs/proposal/performance_monitoring.md

```diff
@@ -0,0 +1,82 @@
+---
+title: proposal of Performance Monitoring
+authors:
+- "@skwwwwww" # Authors' GitHub accounts here.
+reviewers:
+- ""
+- TBD
+approvers:
+- ""
+- TBD
+
+creation-date: 2024-09-21
+
+---
+
+## proposal of Kmesh observability
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+### Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap.
+
+A good summary is probably at least a paragraph in length.
+-->
+
+The significance of performance monitoring in Kmesh as the foundation for efficient, scalable, and robust grid systems is paramount. In Kmesh, monitoring key metrics such as Kmesh_deamon activity, the number of eBPF maps, the number of entries in each map, and the duration of eBPF method executions is crucial to ensure optimal system performance.
+
+In this proposal, I will analyze the performance monitoring metrics of Kmesh. I will also suggest the implementation of enhanced observability features within Kmesh to capture these critical performance indicators. This will allow users to seamlessly monitor Kmesh's performance and ensure system efficiency.
```

Comment:
Does this proposal consider control plane monitoring, like record the time used to handle xds response

Reply:
Currently, only the monitoring of kmesh resource usage,eBPF map-related metrics, and the execution time of eBPF functions have been considered.